android/src/main/java/com/google/android/react/navsdk/NavViewManager.java

```diff
@@ -253,19 +268,19 @@ public void receiveCommand(@NonNull FrameLayout root, String commandId, @Nullabl
         getFragmentForRoot(root).animateCamera(args.getMap(0).toHashMap());
         break;
       case SET_TRAFFIC_INCIDENT_CARDS_ENABLED:
-        getFragmentForRoot(root).setTrafficIncidentCardsEnabled(args.getBoolean(0));
+        getNavFragmentForRoot(root).setTrafficIncidentCardsEnabled(args.getBoolean(0));
```

Comment:
formatting (although we will run formatting task as soon as we get this and auto landed, let's have clean code anyway)